fuzz/cli.rs

```diff
@@ -160,13 +161,74 @@ fn run(fuzzer: Fuzzer, target: &str) -> Result<(), Error> {
     }
 }
 
+/// Get input seeds for fuzz target
+fn get_seed_dir(target: &str) -> PathBuf {
+    let seeds_dir = SEED_ROOT.join(target);
+    if seeds_dir.exists() {
+        seeds_dir
+    } else {
+        SEED_ROOT.join("default")
+    }
+}
+
+/// Create corpus dir for fuzz target
+fn create_corpus_dir(base: &PathBuf, target: &str) -> Result<PathBuf, Error> {
```

Comment:
How about `base: impl AsRef<Path>`? Then your calling to `create_corpus_dir` can be simplified.